cost/azure/unattached_volumes/azure_delete_unattached_volumes.pt

```diff
@@ -370,7 +373,15 @@ script "js_filter_activities", type: "javascript" do
   result "results"
   code <<-EOS
   var results = []
-  var activities = ds_azure_log_activity;
+  _.each(ds_filter_disks, function(disk, diskIndex) {
+    ds_filter_disks[diskIndex].activities = []
+    _.each(ds_azure_log_activity, function(activity, actIndex){
+      if (activity.resourceID === disk['disk_id']) {
+        ds_filter_disks[diskIndex].activities.push(activity)
+      }
+    })
```

Comment:
This might be simpler using a filter?
Example:
```javascript
    ds_filter_disks[diskIndex].activities = ds_azure_log_activity.filter(function(activity) {
      return activity.resourceID === disk['disk_id']
    }
```